### DISCUSSION TITLE
Starting "Programs and Features" destroys existing a explorer window.

### DISCUSSION THREAD

**Discussion by kagekiyo7** [Open-Shell/Open-Shell-Menu#1845] [Q&A] [answered]
As the title says.
The single explorer window that is already running will disappear and be replaced by a new "Programs and Features" window.
I launched the Programs and Features directly using a shortcut rather than from the Control Panel.
This was happening with the Classic Shell as well.
Does this not usually happen?
windows10 64bit

**Comment by ge0rdi** [comment]
For me it seems to open `Programs and Features` in new window.

I have typed `progra` into search and then selected `Programs and Features` item from search results (`Control Panel` section).

**Reply by kagekiyo7** [reply]
hmm, I followed the procedure and recorded it.
https://www.youtube.com/watch?v=-jXW77IuwaM

**Reply by ge0rdi** [reply]
Is the behavior different if you go to `Apps & features` in Windows `Settings` and click on `Programs and Features` link there?

**Reply by kagekiyo7** [reply]
No different. I tried it, The explorer window was destroyed as well.

**Reply by ge0rdi** [reply, marked as answer]
Thank you for confirming.
So the issue is not related to `Open-Shell` (you can even double check by uninstalling `Open-Shell`).

I have no clue why it behaves the way you are observing.
Maybe it is caused by some third-party software?

https://answers.microsoft.com/en-us/windows/forum/all/win-10-pro-control-panel-opens-in-explorer-instead/76a01ab8-6a7c-4a52-b7e0-ce2ab6e34d61
This looks similar. Do you use `QTTabBar` by any chance?

**Reply by kagekiyo7** [reply]
Oh my god.
That's right. I have installed QTTabBar. I completely forgot.
I don't know why, but I mistakenly thought that the tabbed feature of explorer was due to Open Shell.
I have taken up a lot of your extra time. My apologies.
